File: PrimSkin/app/src/main/java/com/prim/skin/MainActivity.java
package com.prim.skin;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.prim_skin.PrimSkin;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void changeSkin(View view) {
        PrimSkin.getInstance().loadSkin("file:///android_asset/test.skin");
    }

    public void resumeSkin(View view) {
        PrimSkin.getInstance().clearSkin();
    }
}

File: PrimFastCommon/prim_test/src/main/java/com/prim/prim_test/MainActivity.kt
package com.prim.prim_test

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.prim.prim_test.http.TestHttpActivity
import org.jetbrains.anko.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun testPrimHttp(view: View) {
//        var intent = Intent(this,TestHttpActivity::class.java)
        //anko DSL
        startActivity<TestHttpActivity>()
    }
}
